### PR TITLE
Update secret session API in EnactTrust-Veraison API sequence

### PR DESCRIPTION
**Pull request by tomoveu**
This PR makes sure our API diagram is up-to-date.

* Change `response code` from EnactTrust Backend to Agent
* Veraison Verifier extracts `nodeid` and `challenge` from TPMS_ATTEST to have a secret to verify
* Fix `typo` in periodic evidence flow

Signed-off-by: Dimitar Tomov <dimi@tpm.dev>

### CODE REVIEW

**Comment by tomoveu** [comment]
@thomas-fossati I will wait for you review before merging these changes.

File: docs/sequence-diagrams/api-integration.puml
Comment (thomas-fossati) [inline docs/sequence-diagrams/api-integration.puml:27]: `challenge-blob` is computed by BE in the following step.  The nonce in this response body is effectively Veraison's generated "the-challenge"


```suggestion
	V_EviVfy -> BE: Status: 201, Location: /session/123, Body: {"nonce"="the-challenge", ...}
```

**Review by thomas-fossati — COMMENTED** [review]
one small typo, the rest looks great!

**Comment by tomoveu** [comment]
@thomas-fossati I have submitted your proposed change using GitHub web. Please re-review just in case and I will merge.

**Review by thomas-fossati — APPROVED** [review]
thanks!

**Pull request merged**
